build.gradle

```diff
@@ -117,6 +117,30 @@ publishing {
                 }
 
                 developers {
+                    developer {
+                        name = "Sadayuki Furuhashi"
+                        email = "frsyuki@gmail.com"
+                    }
+                    developer {
+                        name = "Muga Nishizawa"
+                        email = "muga.nishizawa@gmail.com"
+                    }
+                    developer {
+                        name = "Satoshi Akama"
+                        email = "satoshiakama@gmail.com"
+                    }
+                    developer {
+                        name = "Dai MIKURUBE"
+                        email = "dmikurube@treasure-data.com"
+                    }
+                    developer {
+                        name = "Shinichi Ishimura"
+                        email = "shiketaudonko41@gmail.com"
+                    }
+                    developer {
+                        name = "Phu Nguyen"
+                        email = "phu@treasure-data.com"
+                    }
```

Comment:
@minidragon88 I was not sure, but it might be the first time to publish an OSS Embulk plugin to Maven Central that you have ever committed. (I didn't think so, but I was not able to find another.)

Do you mind if including your name and this email address in `pom.xml`, published in Maven Central?  If you don't want, I can remove the entry, change the name, or change the email, as you want.

Reply:
Ah, I found your name in https://github.com/treasure-data/embulk-input-jira/blob/v0.3.0/build.gradle

Reply:
Yes, it's fine with me